components/Carousel/Carousel.js

```diff
@@ -32,12 +33,22 @@ class Carousel {
     }
 
     setImage = (index) => {
-        
         Array.from(this.images).forEach(img => {
             img.style.display = 'none';
+            TweenMax.to(img, .2, {width: 0, display: "none"});
         });
         const currentImage = Array.from(this.images)[index];
-        TweenMax.fromTo(currentImage, 1, {width: 0}, {width: "100%", display: "block"});
+        TweenMax.to(currentImage, 1,{width: "100%", display: "block"});
+    }
+    autoShowCarousel = () => {
+        setInterval( () => {
+            if (this.currentImage === this.imageCount - 1) {
+                this.currentImage = 0;
+            } else {
+                ++this.currentImage
+            }
+            this.setImage(this.currentImage)
+        }, 3000) 
     }
 }
```

Comment:
- Chinedu You did amazing job on the sprint challenge. You properly implemented Tabs and Cards components into the project and were able to filter out articles by selecting particular Tab. You also added new articles to test Your components which concludes the MVP requirements 👍

- Great job tackling the stretch goal and working on Carousel functionality. It was implemented properly. Not only that but You added GSAP animations to the project which make whole experience look a lot more "lively". 

- Overall:
  - Tabs and Cards properly implemented (MVP)
  - New Articles added (MVP)
  - Great answers to questions (MVP)
  - Carousel functionality implemented (Stretch)
  - Animation applied (Stretch)
  - Clean Code
  - Fluid git workflow
  - Excellent job, You displayed mastery on the subject 👍